cellxgene_schema_cli/tests/fixtures/examples_validate.py

```diff
@@ -40,7 +40,6 @@
             "CL:0000066",
             "EFO:0009899",
             "MONDO:0100096",
-            "NCBITaxon:9606",
```

Comment:
all of the fixtures here had to be rewritten because:
1. all existing test fixtures had human + mouse data combined, which is now not allowed
2. `organism_ontology_term_id` now moved to `uns`